food-waste-express-api/routes/makeapost.js

```diff
@@ -33,8 +33,14 @@ router.get('/', async (req, res) => {
     try {
         const query = new Parse.Query("Products")
         query.descending("createdAt")
-        products = await query.find()
-  
+        allProducts = await query.find()
+        let products = []
+
+        allProducts.map((product) => {
+            if (!product.get("placedOnHoldBy")) {
```

Comment:
If you are using this input a lot of places maybe we can consider making it a global variable